imgui/__init__.py

```diff
@@ -267,3 +269,24 @@
 INPUT_TEXT_READ_ONLY = core.INPUT_TEXT_READ_ONLY
 #: Password mode, display all characters as '*'
 INPUT_TEXT_PASSWORD = core.INPUT_TEXT_PASSWORD
+
+
+# extra aliases for backwards compatibility
+def _core_to_extra_deprecated(fn):
+    return _compat.deprecated("""
+        The imgui.core.{0}() is deprecated
+        and will be removed in 1.0.0 version of pyimgui.
+        Please use imgui.extra.{0}()
+        or imgui.{0}() instead.
+        """.format(fn.__name__)
+    )(fn)
+
+
+core.font = _core_to_extra_deprecated(extra.font)
+core.styled = _core_to_extra_deprecated(extra.styled)
+core.istyled = _core_to_extra_deprecated(extra.istyled)
+core.vertex_buffer_vertex_pos_offset = _core_to_extra_deprecated(extra.vertex_buffer_vertex_pos_offset)  # noqa
+core.vertex_buffer_vertex_uv_offset = _core_to_extra_deprecated(extra.vertex_buffer_vertex_uv_offset)  # noqa
+core.vertex_buffer_vertex_col_offset = _core_to_extra_deprecated(extra.vertex_buffer_vertex_col_offset)  # noqa
+core.vertex_buffer_vertex_size = _core_to_extra_deprecated(extra.vertex_buffer_vertex_size)  # noqa
+core.index_buffer_index_size = _core_to_extra_deprecated(extra.index_buffer_index_size)  # noqa
```

Comment:
Random question, what does the `# noqa` comment mean?

Reply:
This common mark for linters like `pep8` (and many IDEs) to exclude current line from style checking.  This line is longer than 79 columns but splitting into multiple lines would IMHO reduce readability.

Such things are already my habit. Now I see that project doesn't have linting in build pipeline. This is oversight. I will add another stage to Travis CI config so all `# noqa` will have a bit more sense.